centosstream_test/mockbuild.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e
+set -x
+
+if ! groups | grep mock; then
+	echo 'No mock group membership; be sure to install mock'
+	echo 'and add the current user to the "mock" group'
+	exit 1
+fi
+
+TARGET=$1
```

Comment:
Looks good. Can we change the name TARGET to something like DIST_RELEASE ?

Reply:
Done.  Though, now that it tests both Fedora and CentOS Stream, should I rename the directory to `mockbuild_test`?

Reply:
Yes, makes sense.

Reply:
OK.  Since this ultimately changes the commit message, directory, etc., I felt like this was a good time to rebase, and continue in a new PR.